docs/cloud/pouta/images.md

```diff
@@ -88,6 +88,12 @@ do not want to use CentOS. Note that Ubuntu-18.04 is that one image that
 has `ubuntu` as the default username instead of `cloud-user`, since it was the
 first image type we started to provide directly from the upstream.
 
+### Ubuntu-20.04 LTS
+We will provide Ubuntu-20.04 LTS images which is the latest in Ubuntu image family for who
```

Comment:
Using future tense sounds weird, I would suggest trimming this to:

```
-We will provide Ubuntu-20.04 LTS images which is the latest in Ubuntu image family for who
+Ubuntu-20.04 LTS image is the latest in Ubuntu image family for those who
```

(also `those` added)

Reply:
There are some grammatical fixes.